core-api/ormconfig.json

```diff
@@ -5,6 +5,8 @@
     "username": "postgres",
     "password": "postgres",
     "database": "core",
+    "syncronized": true,

```

Comment:
Why is it necessary? If we're using migrations to control our schema, `synchronize` (yep, I think you misspelled it) shouldn't be here.

Reply:
Actually, I don't know. I will remove that change.

Reply:
Done